_adr/82/index.adoc

```diff
@@ -0,0 +1,87 @@
+---
+num: 82
+title: Streaming Unit Hour Metrics
+status: "Draft"
+authors:
+  - "Peter Braun"
+tags:
+  - "kafka"
+#applies_padrs: # What PADRs does this ADR apply?
+#applies_patterns: # What APs does this ADR apply?
+---
+
+## Context and problem statement
+
+Streaming Unit pricing will be accomplished by introducing a new concept called ‘Streaming Unit Hours’. This is defined as Cluster Hours multiplied by the Streaming Unit Size. Here Streaming Unit Size should not be confused with Quota usage which is required for prepaid pricing.
```

Comment:
Conceptually there's a difference between passing a number that is _defined_ to be the number of streaming units, and passing a factor/coefficient that _happens to be_ the number of streaming units. The former makes linearity of cost something that's built into the architecture. The latter does not. While we think that's fine right now (i.e kind of a built-in assumption of streaming units) we may find that as we scale up we want to discount larger clusters. For example, a 3 broker cluster also needs three zk/kraft controller nodes, but a 12 broker cluster still only needs three zk/kraft controller nodes, so the VM _costs_ do not scale linearly, even if what we charge may do.

Reply:
The discount could come from the control plane, by setting the number to e.g. `2.5` instead of `3`. You're right that his shouldn't be called streaming hours then. How about `billed_streaming_units`?